crates/db_views/src/custom_emoji_view.rs

```diff
@@ -35,16 +35,49 @@ impl CustomEmojiView {
     }
   }
 
-  pub async fn get_all(
+  pub async fn get_all(pool: &mut DbPool<'_>) -> Result<Vec<Self>, Error> {
```

Comment:
You can probably get rid of this one now. Or leave it as a helper function, but have it call `self.list(.... true)